### DISCUSSION TITLE
为什么搜索网站的图标没了，是不是极简模式的bug问题

### DISCUSSION THREAD

**Discussion by yurusama-1** [xiaoshuapp/explorer-xiaoshu#579] [Feedback]
为什么搜索网站的图标没了，是不是极简模式的bug问题

**Comment by nexmoe** [comment]
图标抓取服务down掉了，找机会修复一下

**Reply by zhonglixun** [reply]
> 图标抓取服务down掉了，找机会修复一下

修复了吗？我好像还是加载不出来。那些很常见的网站图标不能直接存下来在本地吗？反正10年都不见得会换图标。每次都要抓取反而费劲不是？

**Reply by HolmesZ** [reply]
1.0.3版本修复了这个问题，但只有chrome的扩展是最新版的，edge和firefox的扩展还是0.9.16。可以麻烦作者同步一下版本吗？
